### PR TITLE
Change Ballerina grammar source

### PR DESCRIPTION
**Pull request by Shan1024**
## Description

This PR updates Ballerina language grammar source.

## Checklist:

- [ ] **I am associating a language with a new file extension.**
  - [ ] The new extension is used in hundreds of repositories on GitHub.com
    - Search results for each extension:
      
      - https://github.com/search?utf8=%E2%9C%93&type=Code&ref=searchresults&q=extension%3AFOOBAR+KEYWORDS+NOT+nothack
  - [ ] I have included a real-world usage sample for all extensions added in this PR:
    - Sample source(s):
      - [URL to each sample source, if applicable]
    - Sample license(s):
  - [ ] I have included a change to the heuristics to distinguish my language from others using the same extension.

- [ ] **I am adding a new language.**
  - [ ] The extension of the new language is used in hundreds of repositories on GitHub.com.
    - Search results for each extension:
      
      -  https://github.com/search?utf8=%E2%9C%93&type=Code&ref=searchresults&q=extension%3AFOOBAR+KEYWORDS+NOT+nothack
  - [ ] I have included a real-world usage sample for all extensions added in this PR:
    - Sample source(s):
      - [URL to each sample source, if applicable]
    - Sample license(s):
  - [ ] I have included a syntax highlighting grammar.
  - [ ] I have included a change to the heuristics to distinguish my language from others using the same extension.

- [ ] **I am fixing a misclassified language**
  - [ ] I have included a new sample for the misclassified language:
    - Sample source(s):
      - [URL to each sample source, if applicable]
    - Sample license(s):
  - [ ] I have included a change to the heuristics to distinguish my language from others using the same extension.

- [X] **I am changing the source of a syntax highlighting grammar**
  
  - Old: https://github-lightshow.herokuapp.com/?utf8=%E2%9C%93&scope=source.ballerina&grammar_format=auto&grammar_url=&grammar_text=&code_source=from-url&code_url=https%3A%2F%2Fgithub.com%2Fballerina-platform%2Fballerina-examples%2Fblob%2Fmaster%2Fexamples%2Fhttp-circuit-breaker%2Fhttp_circuit_breaker.bal&code=ddsdsd
  - New: https://github-lightshow.herokuapp.com/?utf8=%E2%9C%93&scope=from-url&grammar_format=auto&grammar_url=https%3A%2F%2Fgithub.com%2Fballerina-platform%2Fballerina-grammar%2Fblob%2Fmaster%2Fsyntaxes%2Fballerina.tmLanguage&grammar_text=&code_source=from-url&code_url=https%3A%2F%2Fgithub.com%2Fballerina-platform%2Fballerina-examples%2Fblob%2Fmaster%2Fexamples%2Fhttp-circuit-breaker%2Fhttp_circuit_breaker.bal&code=

- [ ] **I am adding new or changing current functionality**
  
  - [ ] I have added or updated the tests for the new or changed functionality.

### CODE REVIEW

File: vendor/licenses/grammar/ballerina-grammar.txt [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
```
Comment (pchaigno) [inline vendor/licenses/grammar/ballerina-grammar.txt]: Have you been using the script as per [the contribution guidelines](https://github.com/github/linguist/blob/master/CONTRIBUTING.md#changing-the-source-of-a-syntax-highlighting-grammar) to add this file? It looks weird, but it might be an issue with the script.

File: grammars.yml [hunk 2]
```diff
@@ -10,7 +10,7 @@ vendor/grammars/Agda.tmbundle:
 vendor/grammars/Alloy.tmbundle:
 - source.alloy
 vendor/grammars/Assembly-Syntax-Definition:
-- source.assembly.unix
+- source.x86
```
Comment (pchaigno) [inline grammars.yml]: There are a several changes in this file that look unintentional and unrelated to this pull request.

File: vendor/licenses/grammar/ballerina-grammar.txt (hunk 1 above)
Comment (Shan1024) [reply vendor/licenses/grammar/ballerina-grammar.txt]: Yes, I used the script to update the grammar.

File: grammars.yml (hunk 2 above)
Comment (Shan1024) [reply grammars.yml]: Changes were done by the script. But I'll undo these and commit again.

**Comment by Shan1024** [comment]
@pchaigno - URLs updated and requested changes were done.

**Comment by Shan1024** [comment]
@pchaigno - Earlier, the ballerina entry in the `.gitmodules` were added to the end of the file by the script and were causing a test failure. I have updated the location of the Ballerina entry in the `.gitmodules` file. But still the `TestPedantic#test_submodules_are_sorted` fails. Seems like the entry is in the correct place now. Do you have any idea why is this?

Earlier, the submodule name was `language-ballerina` and now it was changed to `ballerina-language`. Can this cause an issue?

**Comment by pchaigno** [comment]
That test relies on the [sort-submodules](https://github.com/github/linguist/blob/14a7cb2d1b3d6f822701bccf36666303509c7621/script/sort-submodules) script. You can actually use that script to sort `.gitmodules` entries for you.

**Comment by Shan1024** [comment]
@pchaigno - I ran the script. But it removed the submodule entry completely 😢

**Comment by Shan1024** [comment]
@pchaigno - All issues resolved. Please review and merge.

File: vendor/licenses/grammar/matlab.tmbundle.txt [hunk 3]
```diff
@@ -1,15 +0,0 @@
----
```
Comment (pchaigno) [inline vendor/licenses/grammar/matlab.tmbundle.txt]: Could you revert this change? It looks unrelated to this pull request.

**Comment by Shan1024** [comment]
@pchaigno - Done 😃

**Comment by pchaigno** [comment]
Great! One last thing: could you update the Lightshow examples? It looks like it can't find the code to highlight.

**Comment by Shan1024** [comment]
@pchaigno - Lightshow examples updated.
